packages/shared/src/lib/featureManagement.ts

```diff
@@ -41,6 +42,7 @@ const feature = {
   copyLink: new Feature('copy_link', false),
   onboardingOptimizations: new Feature('onboarding_optimizations', false),
   userAcquisition: new Feature('user_acquisition', UserAcquisition.Control),
+  feedAdSpot: new Feature('feed_ad_spot', FeedAdSpot.Control),
```

Comment:
@rebelchris just fyi for gb setup/naming

Reply:
@capJavert Turned it on with number value, this is a first time I think we have numbers. can you just validate on a preview domain? (with the gb tools)

I remember that flagsmith had some issues with numbers.

Reply:
Yep, will do now.

Reply:
Confirmed it works.